### PR TITLE
Category panel

### PR DESCRIPTION
**Pull request by ryayak1460**
The following changes since commit 4455a4acca2056b3a9c721e69bddd60cd1b6afe1:

  Merge pull request #93 from wbprice/64-results-page-style (2016-04-12 12:20:45 -0400)

are available in the git repository at:

  https://github.com/ryayak1460/okcandidate.git category-panel

for you to fetch changes up to 6cc7f57db3e337f6c6f7a9a641dd422d86dc138d:

  Allow the badge to grow itself (2016-04-12 15:53:45 -0400)

----------------------------------------------------------------
Ryan Y (4):
      Provide collapsible categories
      Include dropdown icon and flexible container
      Allow badge to accept external styles
      Allow the badge to grow itself

 .../molecules/CandidateMatchProfileBadge.jsx       |   8 +-
 .../organisms/CandidateMatchCandidate.jsx          |  51 ++++++---
 .../molecules/CandidateMatchProfileBadgeSpec.js    | 105 +++++++++++++++++
 .../organisms/CandidateMatchCandidateSpec.js       | 126 +++++++++++++++++++++
 4 files changed, 270 insertions(+), 20 deletions(-)
 create mode 100644 test/frontend/components/molecules/CandidateMatchProfileBadgeSpec.js
 create mode 100644 test/frontend/components/organisms/CandidateMatchCandidateSpec.js

### CODE REVIEW

**Comment by ryayak1460** [comment]
Affects #94.

File: assets/js/components/organisms/CandidateMatchCandidate.jsx [hunk 1]
```diff
@@ -14,27 +15,40 @@ class CandidateMatchCandidate extends Component {
...
-                questions={category.questions} />
-            )
-          })
-          :
-          null
-        }
+        <div style={{display: 'flex'}}>
+          <CandidateMatchProfileBadge
+            candidateName={this.props.candidateName}
+            compositeMatchScore={this.props.compositeMatchScore} />
```
Comment (wbprice) [inline assets/js/components/organisms/CandidateMatchCandidate.jsx]: To get the profile badge to fill the space and push the dropdown button to the right, pass a style object in props here.  You'll need to modify CandidateMatchProfileBadge to accept external styles though.  https://github.com/Code4HR/okcandidate/blob/master/assets/js/components/atoms/CandidateMatchRating.jsx#L35-L47

Comment (ryayak1460) [inline assets/js/components/organisms/CandidateMatchCandidate.jsx]: c5d16dd allows external styles on the badge.

**Comment by wbprice** [comment]
**Style Feedback **
screen shot 2016-04-12 at 5 16 13 pm 
Can you remove the gray border of the collapsible panel?  Should be able to override this with CSS.

screen shot 2016-04-12 at 5 18 01 pm
When the panel is open, can you use the glyphicon that faces up?

File: assets/js/components/organisms/CandidateMatchCandidate.jsx [hunk 2]
```diff
@@ -14,27 +15,41 @@ class CandidateMatchCandidate extends Component {
...
-          :
-          null
-        }
+        <div style={{display: 'flex'}}>
+          <CandidateMatchProfileBadge
+            candidateName={this.props.candidateName}
+            compositeMatchScore={this.props.compositeMatchScore}
+            style={{flex: 1}} />
+          <Button onClick={event => this.toggleCategories()}>
+            <Glyphicon glyph="chevron-down" />
```
Comment (wbprice) [inline assets/js/components/organisms/CandidateMatchCandidate.jsx]: You can use an inline ternary here to pick the icon displayed.  If `this.state.showCategory` is true, the icon `chevron-up`should be displayed. If false, display `chevron-down`.

**Comment by wbprice** [comment]
LGTM

**Pull request merged**